bindings/python/core_types/src/py_number.cpp

```diff
@@ -0,0 +1,35 @@
+#include "py_core_types/py_core_types.h"
+
+PyDaqIntf<daq::INumber> declareINumber(pybind11::module_ m)
+{
+    return wrapInterface<daq::INumber>(m, "INumber");
+}
+
+void defineINumber(pybind11::module_ m, PyDaqIntf<daq::INumber> cls)
+{
+    cls.doc() = "Represents either a float or an int number.";
+
+    cls.def("__float__",
```

Comment:
I'm not sure if these two functions are strictly needed since `IBaseObject` already defines
```c++
        .def("__int__", [](daq::IBaseObject* obj) { return static_cast<daq::Int>(daq::BaseObjectPtr::Borrow(obj)); })
        .def("__float__", [](daq::IBaseObject* obj) { return static_cast<daq::Float>(daq::BaseObjectPtr::Borrow(obj)); })
```
but it might not hurt? Please advise.

Reply:
My suggestion is to fix this properly by conversion to native types, but we could also merge this now just for the sake of avoiding issues with `pybind11_stubgen` etc.